praw/models/reddit/emoji.py

```diff
@@ -0,0 +1,81 @@
+"""Provide the EMOJI class."""
+from ...const import API_PATH
+from ..listing.generator import ListingGenerator
+from .base import RedditBase
+from .redditor import Redditor
+
+
+class Emoji(RedditBase):
+    """An individual Emoji object."""
+
+    __hash__ = RedditBase.__hash__
+
+    @property
+    def mod(self):
+        """Provide an instance of :class:`.WikiPageModeration`."""
+        if self._mod is None:
+            self._mod = EmojiModeration(self)
+        return self._mod
+
+    def __init__(self, reddit, subreddit, name, revision=None, _data=None):
+        """Construct an instance of the WikiPage object.
+
+        :param revision: A specific revision ID to fetch. By default, fetches
```

Comment:
Is the revision argument needed?

Reply:
Apologies, a few of these are simply things I didn't catch at 1 AM.